src/createDIContainer.js

```diff
@@ -70,16 +71,14 @@ const enableCoreQuorumsTaskFactory = require('./listr/tasks/setup/local/enableCo
 const startGroupNodesTaskFactory = require('./listr/tasks/startGroupNodesTaskFactory');
 const buildServicesTaskFactory = require('./listr/tasks/buildServicesTaskFactory');
 
-async function createDIContainer(options) {
+async function createDIContainer() {
   const container = createAwilixContainer({
     injectionMode: InjectionMode.CLASSIC,
   });
 
   /**
    * Config
    */
-  const homeDirPath = options.DASHMATE_HOME_DIR ? options.DASHMATE_HOME_DIR : path.resolve(os.homedir(), '.dashmate');
-
   container.register({
     homeDirPath: asValue(homeDirPath),
```

Comment:
It's probably better to remove this value from the container and use constants where it was used.